plugin/listtoggle.vim

```diff
@@ -46,39 +41,22 @@ execute "nnoremap " . s:unique . " <silent> " .
 command! QToggle call <sid>QListToggle()
 command! LToggle call <sid>LListToggle()
 
-
 function! s:LListToggle()
-  if exists("s:quickfix_buffer_number")
+    let buffer_count_before = len(filter(range(1, bufnr('$')), 'buflisted(v:val)'))
```

Comment:
Seems like `len(...)` expression can be pulled out into a function. Seems to be copy-pasted four times.